.jules/bolt.md

```
@@ -0,0 +1,4 @@
+
+## 2024-04-28 - NumPy Percentile Array Optimization
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)